torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -38,6 +43,134 @@
     _linear_args = [Arg() for _ in range(6)]
     _conv_transpose_args = [Arg() for _ in range(11)]
 
+    class MkldnnDeviceOpBase:
+        def get_linear_transpose_weight(self, weight_node):
```

Comment:
`def get_linear_transpose_weight(self, weight_node): -> torch.fx.Node` is this legal syntax?

Reply:
Yes, thanks for your reminder.